packages/core/src/client/sitecore-client.ts

```diff
@@ -246,13 +259,15 @@ export class SitecoreClient implements BaseSitecoreClient {
   protected errorPagesService: ErrorPagesService;
   protected componentService: ComponentLayoutService;
   protected sitePathService: SitePathService;
+  protected graphQLClient: GraphQLClient;
 
   /**
    * Init SitecoreClient
    * @param {SitecoreClientInit} initOptions initOptions for the client, containing site and Sitecore connection details
    */
   constructor(protected initOptions: SitecoreClientInit) {
     this.clientFactory = this.getClientFactory();
+    this.graphQLClient = this.getGraphQLClient();
```

Comment:
You can initialize `graphQLClient` right here. Also, let's add http `debugger` as well.
I believe that `getGraphQLClient` method is not really needed since the same global settings are passed in `getClientFactory()`, right?

```suggestion
    this.graphQLClient = this.clientFactory({
      debugger: debug.http
    })
```

Reply:
ohh yes you're right